### PR TITLE
feat: change labels from howtos to projects

### PR DESCRIPTION
**Pull request by benfurber**
## PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)
- [x] - Unit and e2e tests for the changes that have been added (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Feature (adds functionality)

## What is the current behavior?

User things are named 'How to', 'how-tos', etc.

## What is the new behavior?

Top level is called 'Library', children are called 'projects'. This change is for labels and urls only. Developer things are still called howtos until we migrate the DB.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Git Issues

Part of #3989

### CODE REVIEW

File: src/routes/_.how-to.$slug._index.tsx [hunk 1]
```diff
@@ -1,50 +1,16 @@
...
-  if (!howto) {
-    return []
-  }
-
-  const title = `${howto.title} - How-to - ${import.meta.env.VITE_SITE_NAME}`
-
-  return generateTags(title, howto.description, howto.cover_image?.downloadUrl)
-})
-
 export default function Index() {
```
Comment (mariojsnunes) [inline src/routes/_.how-to.$slug._index.tsx:14]: I think this can be removed

**Comment by davehakkens** [comment]
thought I left a comment before but dont see it anymore.
Question: Is there a conscious reason on why you choose different wording from the issue?

**Comment by benfurber** [comment]
> thought I left a comment before but dont see it anymore. Question: Is there a conscious reason on why you choose different wording from the issue?

Yes. Definitely went with what seemed most natural to me personally. The principle I tried to go with wast a `project` is the main unit, the main 'thing' that should be labelled and so only went with library when talking about all the projects collectively. But very happy to debate the grammar.

**Comment by benfurber** [comment]
@davehakkens See your point. I'll update. Applying your approach but with some suggestions: 

Create button
"Add your how-to" -> "Add to the library" or "Add your project to the library"

Create page
"Create a how-to" -> "Add to the library" or "Add your project to the library"

Profiles/Contribution tab: 
"How-to's" -> "Library Projects"


Thoughts?

**Comment by davehakkens** [comment]
Tried that before but adding 'project' to the wording often makes it to long for the layout making it look strange

**Comment by benfurber** [comment]
@davehakkens Have a look [at this now](https://github.com/ONEARMY/community-platform/pull/4027/commits/dd5d3d956aec497e6f3c3472459931a7a314d1aa).

**Comment by davehakkens** [comment]
The url doest go anywhere. And the preview doesnt show any changes at first glance, not sure if i'm seeing the latest?
image

**Comment by benfurber** [comment]
@davehakkens Actually freshly deployed now: https://community-platform-pr-4027.fly.dev/library?sort=Newest

**Comment by davehakkens** [comment]
I see the same as the screenshot above. Is that the latest?

**Comment by benfurber** [comment]
@davehakkens If you refresh and everything you don't see this? 
Screenshot 2024-12-13 at 10 06 15

**Comment by davehakkens** [comment]
Nice works for me, two changes that would be nice:

1. Change text in searchbar
image

2. Change text in the header
image

**Comment by benfurber** [comment]
One done. The second is an environement variable so I'll change the default option but the actual prod variables will need changing when this is deployed.

**Pull request merged**